tests/e2e/common.go

```diff
@@ -70,6 +72,15 @@ type OsmTestData struct {
 	cleanupNamespaces map[string]bool
 }
 
+// GetEnv is a wrapper for Getenv, that returns a default value instead of empty when `envVar` is not defined
+func GetEnv(envVar string, defaultValue string) string {
+	val := os.Getenv(envVar)
+	if val == "" {
+		return defaultValue
+	}
+	return val
+}
+
```

Comment:
This is implemented in at lease one other file: https://github.com/openservicemesh/osm/blob/c98e2556ca03ec7a0774e38066ece4a16f53ebb2/demo/cmd/common/books.go#L97

Might as well move this under `pkg/utils` and use that.

Reply:
let me try. I hope to not run in cyclic imports.